net/unbound/Makefile

```diff
@@ -122,14 +122,14 @@ endef
 
 define Build/InstallDev
 	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/unbound.h $(1)/usr/include/
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/include/unbound.h $(1)/usr/include/
 	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libunbound.{so*,a,la} $(1)/usr/lib/
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/lib/libunbound.{so*,a,la} $(1)/usr/lib/
```

Comment:
`.so` files are expected to have executable bit set and `INSTALL_DATA` will remove them I guess.  So this part might be wrong.

For dev files in `PKG_INSTALL_DIR`, using `$(CP)` should be just fine as it has the archive flag set.